clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/model/KubernetesJobStatus.groovy

```diff
@@ -40,6 +44,8 @@ class KubernetesJobStatus implements JobStatus, Serializable {
   Set<String> securityGroups
   @JsonIgnore
   Pod pod
+  @JsonIgnore
```

Comment:
I wonder if this should be split into two classes that implement a `KubernetesJobStatus` interface...rather than have a pair of fields where exactly one has to be set (and the have the logic check which is null).